amulet/helpers.py

```diff
@@ -67,7 +67,7 @@ def juju(args, env=None, include_model=True):
         else:
             args = list(args)
             # insert the model arg after the command, but before any other args
-            args[1:0] = [model_flag, default_environment()]
+            args += [model_flag, default_environment()]
```

Comment:
The model flag can't come at the end for some commands; specifically `juju scp`, which the original change was done to fix:

```
[johnsca@murdoch:~/juju/python-libjuju] lxd:admin/controller:1 $ juju scp -m lxd:admin/default tox.ini test/0:
[johnsca@murdoch:~/juju/python-libjuju] lxd:admin/controller:1 $ juju scp tox.ini test/0: -m lxd:admin/default
ERROR unit "test/0" not found
```

We might end up needing to  encode a specific list of compound commands for 1.x.  :(